e2e-tests/operator-self-healing/conf/no-limits.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: pxc.percona.com/v1
+apiVersion: pxc.percona.com/v1-6-0
```

Comment:
please dont change files in `/conf/` folders, automatically adjust value by `cat_config` function